src/lightning/fabric/plugins/collectives/torch_collective.py

```diff
@@ -30,6 +30,12 @@ def __init__(self) -> None:
             raise RuntimeError("Torch distributed is not available.")
         super().__init__()
 
+    @property
+    def group(self) -> CollectibleGroup:
```

Comment:
lets add dostring

Reply:
The other methods also don't have docs (see super class). We can leave this for the future once this becomes a feature. Right now it is experimental, and likely to change (like in this PR).